plugins/modules/ntnx_pc_unregistration_v2.py

```diff
@@ -0,0 +1,245 @@
+            "number_of_subtasks": 0,
+            "operation": "UnregisterPC",
+            "operation_description": "Unregister Prism Central",
+            "owned_by": {
+                "ext_id": "00000000-0000-0000-0000-000000000000",
+                "name": "admin"
+            },
+            "parent_task": null,
+            "progress_percentage": 100,
+            "root_task": null,
+            "started_time": "2025-01-29T06:51:20.392266+00:00",
+            "status": "SUCCEEDED",
+            "sub_steps": [
+                {
+                    "name": "Unregistering cluster started"
+                },
+                {
+                    "name": "Precheck completed successfully"
+                },
+                {
+                    "name": "Successfully unconfigured entities"
+                },
+                {
+                    "name": "Successfully revoked trust"
+                },
+                {
+                    "name": "Unregister cluster 86b54161-1221-1233-9875-89afcd365004 completed successfully"
+                }
+            ],
+            "sub_tasks": null,
+            "warnings": null
+        }
+task_ext_id:
+    description: External ID of the task.
+    type: str
+    returned: always
+    sample: "ZXJnb24=:7f0399f6-370c-59f8-b7a3-c50e4b91a6d0"
+
+pc_ext_id:
+    description: External ID of the local cluster.
+    type: str
+    returned: always
+    sample: "18553f0f-8547-4115-9696-2f698fbe7117"
+
+changed:
+    description: This indicates whether the task resulted in any changes
+    type: bool
+    returned: always
+    sample: true
+
+error:
+    description: Error message if any.
+    type: str
+    returned: When an error occurs
+    sample: null
+"""
+
+import traceback  # noqa: E402
+import warnings  # noqa: E402
+
+from ansible.module_utils.basic import missing_required_lib  # noqa: E402
+
+from ..module_utils.base_module import BaseModule  # noqa: E402
+from ..module_utils.utils import remove_param_with_none_value  # noqa: E402
+from ..module_utils.v4.prism.helpers import get_pc_config  # noqa: E402
+from ..module_utils.v4.prism.pc_api_client import (  # noqa: E402
+    get_domain_manager_api_instance,
+    get_etag,
+)
+from ..module_utils.v4.prism.tasks import wait_for_completion  # noqa: E402
+from ..module_utils.v4.spec_generator import SpecGenerator  # noqa: E402
+from ..module_utils.v4.utils import (  # noqa: E402
+    raise_api_exception,
+    strip_internal_attributes,
+)
+
+SDK_IMP_ERROR = None
+try:
+    import ntnx_prism_py_client as prism_sdk  # noqa: E402
+except ImportError:
+    from ..module_utils.v4.sdk_mock import mock_sdk as prism_sdk  # noqa: E402
+
+    SDK_IMP_ERROR = traceback.format_exc()
+
+# Suppress the InsecureRequestWarning
+warnings.filterwarnings("ignore", message="Unverified HTTPS request is being made")
+
+
+def get_module_spec():
+    module_args = dict(
+        pc_ext_id=dict(type="str", required=True),
```

Comment:
Make attribute names more explainable. Its very hard to understand what they mean from `pc_ext_id` and `ext_id`